### DISCUSSION TITLE
Export embroidery file without dialog

### DISCUSSION THREAD

**Discussion by chris28xD** [inkstitch/inkstitch#2493] [Q&A]
Hey there,
I want to export the embroidery file (for example .pes) via a shortcut or a button, so without going through the save as dialog.
I found Inkscape brings the "--export-type=" attribute, but .pes exports don't seem to work.
Is there anything I'm missing out?

**Comment by kaalleen** [comment]
This doesn't work with the current official release.

But should work with development releases up from this one here:
https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kaalleen-add_to_inkscape_export
